### ISSUE TITLE
Inconsistent JSON When Using Units With Logical DIsk API

### ISSUE THREAD

**Issue by antonc42** [NagiosEnterprises/ncpa#250]
When using the units option on a logical disk, the JSON returned shows the units differently than when no units are specified and the default (bytes) is used. The default shows the value as the first array element and the units as the second. When requesting units, the first array element is instead another one-element array containing the value while the units are still the second array element.

With units (`/api/disk/logical/|?token=token&units=K`):
```json
      "total_size": [
        [
          30335164.416
        ], 
        "Kb"
      ], 
      "used": [
        [
          1213370.368
        ], 
        "Kb"
      ], 
      "free": [
        [
          29121794.048
        ], 
        "Kb"
      ], 
```

Without units (`/api/disk/logical/|?token=token`):
```json
      "total_size": [
        30335164416, 
        "b"
      ], 
      "used": [
        1213370368, 
        "b"
      ], 
      "free": [
        29121794048, 
        "b"
      ], 
```

OS: CentOS 7 x86_64
NCPA: v1.8.1 via the `ncpa-1.8.1-1.el7.centos.x86_64.rpm` package

**Comment by jomann09** [comment]
I noticed this when I was working on updating the `units` a little while ago. I had thought I put something in to remove the list when there was only one value but I may not be remembering correctly. I agree that this is inconsistent and shouldn't do this. It will definitely be fixed for 2.0.0.

**Comment by antonc42** [comment]
Thanks!

**Comment by jomann09** [comment]
As an update (for you and for me later) it does look like this was supposed to be fixed in 2.0.0 from this bug fix I found in the CHANGES.md file.

Fixed single value objects that have been updated via units (K, M, G, T) from becoming lists in the API

I'll look into it more a little later today.

**Comment by jomann09** [comment]
Fixed this for 2.0.0 !
